### DISCUSSION TITLE
Worn tape (1993)

### DISCUSSION THREAD

**Discussion by Degamisu** [ntsc-rs/ntsc-rs#155] [Presets]
I had a worn tape (TDK EHG) and decided to mimic what it looked like. I had this preset stored for a while and meant to post it a long time ago.

![image](https://github.com/user-attachments/assets/fcd25744-d4b5-4a87-8f47-f82253592fa6)

[Worn-Tape2.json](https://github.com/user-attachments/files/16562244/Worn-Tape2.json)

>[!NOTE]
>I will be posting more presets in my storage a little more often, so expect to see some more!

**Comment by smfthe1199** [comment]
Immediately added to the preset pack that I am updating.
